### PR TITLE
freno-new-fluent-bit-chart

### PR DESCRIPTION
**Pull request by frankreno**
Move Fluent-bit subchart to new location, off of helm/stable.

### CODE REVIEW

File: deploy/helm/fluent-bit-overrides.yaml [hunk 1]
```diff
@@ -26,98 +22,108 @@ env:
...
+        Systemd_Filter  _SYSTEMD_UNIT=var-lib-etcd2.service
+        Max_Entries     1000
+        Read_From_Tail  true
+  outputs: |
+    [OUTPUT]
+        Name          forward
+        Match         *
+        Host          ${CHART}.${NAMESPACE}.svc.cluster.local.
+        Port          24321
+        Retry_Limit False
```
Comment (sumo-drosiek) [inline deploy/helm/fluent-bit-overrides.yaml:121]:
```suggestion
        Retry_Limit   False
```

File: ci/build.sh [hunk 2]
```diff
@@ -76,6 +76,7 @@ if [ -n "$GITHUB_TOKEN" ] && [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
   echo "# This file is auto-generated." > deploy/kubernetes/fluentd-sumologic.yaml.tmpl
   sudo helm init --client-only
   sudo helm repo add falcosecurity https://falcosecurity.github.io/charts
+  sudo helm repo add fluent https://fluent.github.io/helm-charts
```
Comment (sumo-drosiek) [inline ci/build.sh:79]: Should we add it to documentation?

Comment (frankreno) [reply ci/build.sh:79]: Good catch, I will update the appropriate Docs.

**Comment by samjsong** [comment]
These kind of changes to the `values.yaml` feel like a breaking change. If a customer had overriden any fluent-bit section in their `values.yaml` the upgrade could break, right?

**Comment by vsinghal13** [comment]
@frankreno is this ready to merge?

**Comment by frankreno** [comment]
> @frankreno is this ready to merge?

let me actually close this, upgrade to the most recent version, test and resubmit.  This is old enough there may be other changes.  Will do that this week

**Comment by sumo-drosiek** [comment]
Linking related comment: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1052#discussion_r516024737

**Review by sumo-drosiek — CHANGES_REQUESTED** [review]
Remove auto-generated files due to #1052

**Pull request closed without merging**